File: lib/csharp/microsoft.bot.builder.solutions/Skills/SkillResponses.cs
// https://docs.microsoft.com/en-us/visualstudio/modeling/t4-include-directive?view=vs-2017
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Microsoft.Bot.Builder.Solutions.Responses;

namespace Microsoft.Bot.Builder.Solutions.Skills
{
    /// <summary>
    /// Contains bot responses.
    /// </summary>
    public class SkillResponses : IResponseIdCollection
    {
        // Generated accessors
        public const string ErrorMessageSkillError = "ErrorMessageSkillError";
        public const string ErrorMessageSkillNotFound = "ErrorMessageSkillNotFound";
    }
}

File: sdk/typescript/libraries/botbuilder-solutions/src/skills/responses/skillResponses.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */

import { IResponseIdCollection } from '../../';
import { join } from 'path';

export class SkillResponses implements IResponseIdCollection {
    public readonly name: string = SkillResponses.name;
    public static readonly pathToResource: string = join(__dirname, 'resources');
    public static readonly errorMessageSkillError: string = 'ErrorMessageSkillError';
    public static readonly errorMessageSkillNotFound: string = 'ErrorMessageSkillNotFound';
    public static readonly confirmSkillSwitch: string = 'ConfirmSkillSwitch';
}
